openapi/paths/api@register@{provider}.yaml

```diff
@@ -32,3 +31,41 @@ post:
     default:
       $ref: ../components/responses/problem.yaml
 
+delete:
+  operationId: mobileDeregister
+  tags: [mobile]
+  summary: Deregister mobile device
+  security: []
+  description: |
+    Removes a registered device from receiving push notifications.    
+  parameters:
+    - name: provider
+      in: path
+      description: |
+        The type of device that is connecting to the service.
+
+        Only "Firebase" is currently supported for deregistration.
+      required: true
+      example: 'Firebase'
+      schema:
+        type: string
+        enum:
+          - Firebase
+  requestBody:
+    $ref: ../components/requestBodies/mobile-deregistration.yaml
+  responses:
+    '200':
+      description: Device successfully deregistered
+      content:
+        text/plain:
+          schema:
+            type: string
+    '404':
```

Comment:
We actually only return 400 here.  There are two reasons for a 400.  One the provider was invalid.  The other the device was not found.  Technically you are correct it should be a 404 in that case.  Looks like we didn't return the correct error code here.

Reply:
Great catch! They were also documented to return a structured error, but the documentation has now been updated to plain text.